aikau/src/main/resources/alfresco/services/DocumentService.js

```diff
@@ -329,7 +325,12 @@ define(["dojo/_base/declare",
          }
          else
          {
-            url = AlfConstants.URL_SERVICECONTEXT + "components/documentlibrary/data/doclist/" + params;
+            url = AlfConstants.URL_SERVICECONTEXT + "components/documentlibrary/data/";
+            if (payload.site)
```

Comment:
The fragment until (and including) data/site/site1 is consumed by the Share data web script surf-doclist.get (URL pattern /components/documentlibrary/data/site/{site}/{webscript}/{params}) while the remainder is captured as the repository web script to call. As a result, the site is contained twice if both tiers are to function correctly.